docs/tutorials/data_management.md

```diff
@@ -236,6 +232,15 @@ fn main() {
 }
 ```
 
+Then, run the following command.
+
+```console
+$ cargo build && icu4x-datagen --cldr-tag latest --icuexport-tag latest --out my-data-dir --format dir --keys-for-bin target/debug/myapp --locales ja
+```
+
+This directory can be read by the `FsDataProvider` from the `icu_provider_fs` crate. You will also need to activate the feature for the chosen syntax on the `icu_provider` crate.
```

Comment:
This works before the code because the keys in the binary are the same for all providers. Having this explanatory statement down here doesn't make sense, it explicitly introduces `FsDataProvider` which we've already implicitly introduced above.

Reply:
It's also inconsistent to generate first for baked, and after for fs.

Reply:
So if i understand correctly, it was a bug that prevented icu4x-datagen to work on zero keys. 

I've updated the doc to include a simple empty app, then change run the `icu4x-datagen` command, and then have the full main with the `mod.rs` included.

Reply:
please move this back up.

Reply:
Done.